test/extensions/secondorder/diag_ggn/diag_ggn_settings.py

```diff
@@ -125,4 +136,101 @@
+    },
+    {
+        "input_fn": lambda: rand(4, 2, 6, 6),
+        "module_fn": lambda: Sequential(
+            Conv2d(2, 3, kernel_size=3, stride=1, padding=1),
+            ReLU(),
+            # skip connection
+            Parallel(
+                ActiveIdentity(),
+                Sequential(
+                    Conv2d(3, 5, kernel_size=3, stride=1, padding=1),
+                    ReLU(),
+                    Conv2d(5, 3, kernel_size=3, stride=1, padding=1),
+                ),
+            ),
+            # end of skip connection
+            MaxPool2d(kernel_size=3, stride=2),
+            Flatten(),
+            Linear(12, 5),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(),
+        "target_fn": lambda: classification_targets((4,), 5),
+        "id_prefix": "branching-convolution",
+    },
+    {
+        "input_fn": lambda: rand(4, 3, 6, 6),
+        "module_fn": lambda: Sequential(
+            Conv2d(3, 2, kernel_size=3, stride=1, padding=1),
+            ReLU(),
+            # skip connection
+            Parallel(
+                ActiveIdentity(),
+                Sequential(
+                    Conv2d(2, 4, kernel_size=3, stride=1, padding=1),
+                    Sigmoid(),
+                    Conv2d(4, 2, kernel_size=3, stride=1, padding=1),
+                    branching.Parallel(
+                        branching.ActiveIdentity(),
+                        Sequential(
+                            Conv2d(2, 4, kernel_size=3, stride=1, padding=1),
+                            ReLU(),
+                            Conv2d(4, 2, kernel_size=3, stride=1, padding=1),
+                        ),
+                    ),
+                ),
+            ),
+            # end of skip connection
+            MaxPool2d(kernel_size=3, stride=2),
+            Flatten(),
+            Linear(8, 5),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(),
+        "target_fn": lambda: classification_targets((4,), 5),
+        "id_prefix": "nested-branching-convolution",
+    },
+]
+###############################################################################
+#                      Branched models - converter                            #
+###############################################################################
+if TORCH_VERSION_AT_LEAST_1_9_0:
+    LOCAL_SETTINGS += [
+        {
+            "input_fn": lambda: ResNet1.input_test,
+            "module_fn": lambda: convert_module_to_backpack(ResNet1().eval()),
+            "loss_function_fn": lambda: ResNet1.loss_test,
+            "target_fn": lambda: ResNet1.target_test,
+            "id_prefix": "ResNet1",
+        },
+        {
+            "input_fn": lambda: rand(ResNet2.input_test),
+            "module_fn": lambda: convert_module_to_backpack(ResNet2().eval()),
+            "loss_function_fn": lambda: ResNet2.loss_test,
+            "target_fn": lambda: rand(ResNet2.target_test),
+            "id_prefix": "ResNet2",
+        },
```

Comment:
Add these other scenarios (and let me know if you think they should live elsewhere as they mainly test the converter):
1) Network with inplace activation
2) Network with parameter-free module used in multiple places
3) Network with `flatten` operation
4) Network with `multiply` operation
5) Network with `add` operation

Reply:
I'm going to add it to resnet_torchvision.py (which should also be renamed then).
Resolve, because this list is also in the PR description.